builder/node.go

```diff
@@ -163,6 +166,45 @@ func (b *Builder) LoadNodes(ctx context.Context, withData bool) (_ []Node, err e
 	return b.nodes, nil
 }
 
+func (n *Node) MarshalJSON() ([]byte, error) {
+	var status string
+	if n.DriverInfo != nil {
+		status = n.DriverInfo.Status.String()
+	}
+	var err string
+	if n.Err != nil {
+		status = "error"
+		err = strings.TrimSpace(n.Err.Error())
+	}
+	var pp []string
+	for _, p := range n.Platforms {
+		pp = append(pp, platforms.Format(p))
+	}
+	return json.Marshal(struct {
+		Name        string
+		Endpoint    string
+		Platforms   []string          `json:",omitempty"`
```

Comment:
I know we haven't been very consistent in these, but wondering if we should include an explicit name to use in the json output; wdyt? i.e., if we explicitly want the output to be starting with an uppercase;

```suggestion
		Platforms   []string          `json:"Platforms,omitempty"`
```